deploy/docker/fs/opt/appsmith/entrypoint.sh

```diff
@@ -105,32 +81,61 @@ init_env_file() {
   set +o allexport
 }
 
-setup_proxy_variables() {
-  export NO_PROXY="${NO_PROXY-localhost,127.0.0.1}"
+setup_proxy_variables
+init_env_file
```

Comment:
Shouldn't these be the other way around?

```suggestion
init_env_file
setup_proxy_variables
```